lms/static/scripts/frontend_apps/components/test/FilePickerApp-test.js

```diff
@@ -135,6 +142,80 @@ describe('FilePickerApp', () => {
     });
   });
 
+  context('when deeplinkingAPI is present', () => {
+    const deeplinkingAPIPath = '/lti/1.3/deeplinking/formfields';
+    const deeplinkingAPIData = { some: 'data' };
```

Comment:
```suggestion
  context('when deepLinkingAPI is present', () => {
    const deepLinkingAPIPath = '/lti/1.3/deep_linking/form_fields';
    const deepLinkingAPIData = { some: 'data' };
```
There's a lot of this in the JS side. I'm not going to point them all out as I'm sure grep is better at this than me!